changes/zh-cn/develop.md

```diff
@@ -14,6 +14,7 @@
 - [[#5245](https://github.com/seata/seata/pull/5245)] 修复不完整的distribution模块依赖
 - [[#5239](https://github.com/seata/seata/pull/5239)] 修复当使用JDK代理时，`getConfig` 方法获取部分配置时抛出 `ClassCastException` 异常的问题
 - [[#5462](https://github.com/seata/seata/pull/5462)] 在RM中使用`@GlobalTransactional`时,如果RM执行失败会抛出`ShouldNeverHappenException`
+- [[#5794](https://github.com/seata/seata/pull/5794)] 解决redis作为注册中心时cluster无法自定义的BUG
```

Comment:
revert this file, add change on following files:
changes/zh-cn/2.0.0.md
changes/en-us/2.0.0.md